src/test/java/io/anserini/kg/freebase/FreebaseNodeTest.java

```diff
@@ -2,10 +2,19 @@
 
 import org.junit.Test;
 import org.openrdf.rio.ntriples.NTriplesUtil;
+import org.openrdf.model.Literal;
+import org.openrdf.model.ValueFactory;
+import org.openrdf.model.impl.SimpleValueFactory;
 
 import static org.junit.Assert.assertEquals;
 
 public class FreebaseNodeTest {
+
+  /**
+   * Simple value factory to parse literals using Sesame library.
+   */
+  private static ValueFactory valueFactory = SimpleValueFactory.getInstance();
+
   @Test
   public void cleanUri() throws Exception {
     assertEquals("fb:m.02mjmr",
```

Comment:
Jimmy wrote this test. It does not require an Exception but there might be a reason why he put it there. @lintool